.github/workflows/trigger-e2e-tests.yml

```
@@ -8,6 +8,37 @@ on:
+      fail-fast: false
+      matrix:
+        version: [ "8.17.1", "7.17.27" ]
+        env: [ docker, eck ]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)